src/models/market_data_history.rs

```diff
@@ -0,0 +1,65 @@
+use chrono::naive::serde::ts_milliseconds;
+use chrono::NaiveDateTime;
+use serde::{Deserialize, Serialize};
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct Bar {
+    pub o: f64,
+    pub c: f64,
+    pub h: f64,
```

Comment:
lets rename all these fields to their full name close, high, low, open, then use `#[serde(alias = "c")]`